Documentation/ReleasePlan.md

```diff
@@ -0,0 +1,44 @@
+# Release Plan
+
+## Versioning strategy
+
+Coverlet is versioned with Semantic Versioning [2.0.0](https://semver.org/#semantic-versioning-200) that states:
+
+```
+Given a version number MAJOR.MINOR.PATCH, increment the:
+
+MAJOR version when you make incompatible API changes,
+MINOR version when you add functionality in a backwards-compatible manner, and
+PATCH version when you make backwards-compatible bug fixes.
+Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
+```
+
+## Release Calendar
+
+We release 3 components as nuget packages:  
+
+**coverlet.msbuild.nupkg**  
+**coverlet.console.nupkg**  
+**coverlet.collector.nupkg**  
+
+We plan 1 release [once per quarter](https://en.wikipedia.org/wiki/Calendar_year) if there is *at least* 1 new commit of source code on master. This release may be a major, minor, or patch version upgrade from the previous release depending on impact to consumers. 
+**We release intermediate packages in case of severe bug or to unblock users.**
```

Comment:
@vagisha-nidhi however I would like to be "relaxed" with rule...if we block someone we'll release asap, could be the case with new collectors.

Reply:
@MarcoRossignoli That sounds like a good plan. Any idea how much time will it take for the beta to be released?

Reply:
cc: @tonerdo

Reply:
@tonerdo Any updates on when the beta will be officially released?
cc: @cltshivash